cypress/support/step_definitions/variable-import.js

```diff
@@ -1,3 +1,8 @@
+given('I unselect all variables', () => {
+  cy.get('.checkAllText').contains('194');
```

Comment:
something more generic maybe? 194 might change frequently

Reply:
hmm, true, will change it!